src/app/todo-list/checkboxList.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Checkbox from '@mui/material/Checkbox';
+
+export default function CheckboxList() {
+    const [checked, setChecked] = React.useState([-1]);
+
+    const handleToggle = (value: number) => () => {
+        const currentIndex = checked.indexOf(value);
+        const newChecked = [...checked];
+
+        if (currentIndex === -1) {
+            newChecked.push(value);
+        } else {
+            newChecked.splice(currentIndex, 1);
+        }
+
+        setChecked(newChecked);
+    };
```

Comment:
1. 建議value這種太中性的字盡量少單獨使用，因為之後會比較難閱讀
2. 下面是我看到過我自己覺得蠻還不錯的寫法，供參考看看

```suggestion
    const handleToggle = (selectTodoId: string | number) => () => {
        setChecked(
            checkedTodoIdList.include(todoId)
                ? checkedTodoIdList.filter((todoId) => selectTodoId !== todoId)
                : [...checkedTodoIdList, selectTodoId]
        );
    };
```